REST-API-Basics/web/src/main/java/com/epam/esm/web/controller/GiftCertificateController.java

```diff
@@ -27,64 +31,38 @@ public GiftCertificateController(GiftCertificateService giftCertificateService)
 
     @GetMapping
     @ResponseStatus(HttpStatus.OK)
-    public List<GiftCertificate> getAll() {
-        return giftCertificateService.getAll();
-    }
-
-    @GetMapping("/with_tags")
-    @ResponseStatus(HttpStatus.OK)
-    public List<GiftCertificateDto> getAllWithTags() {
-        return giftCertificateService.getAllWithTags();
+    public List<GiftCertificate> getAll(
```

Comment:
Can be combined with "/with-tags" endpoint using. If required, "with-tags" can be used as request parameter.